sdk/src/test/java/io/opentelemetry/sdk/trace/SpanBuilderSdkTest.java

```diff
@@ -230,6 +268,37 @@ public void setAttribute_nullAttributeValue() throws Exception {
     assertThat(span.toSpanData().getAttributes()).isEmpty();
   }
 
+  @Test
+  public void setAttribute_nullAttributeValue_afterEnd() throws Exception {
+    Span.Builder spanBuilder = tracerSdk.spanBuilder(SPAN_NAME);
+    spanBuilder.setAttribute("emptyString", "");
+    spanBuilder.setAttribute("nullString", (AttributeValue) null);
```

Comment:
This will not be added anyway. Maybe we can keep only the values that will be really added? This makes the test more concise.

Reply:
You're right! This is pointless since it is covered by another test!